lib/query-interface.js

```diff
@@ -475,7 +475,7 @@ class QueryInterface {
       // Query generators that use information_schema for retrieving table info will just return an empty result set,
       // it will not throw an error like built-ins do (e.g. DESCRIBE on MySql).
       if (_.isEmpty(data)) {
-        return Promise.reject(`No description found for "${tableName}" table. Check the table name and schema; remember, they _are_ case sensitive.`);
+        return Promise.reject(new Error(`No description found for "${tableName}" table. Check the table name and schema; remember, they _are_ case sensitive.`));
```

Comment:
Nit: you can make this a `throw` since this is a promise context.

Reply:
Yes, throw will be clean